src/test/scala/cromwell/engine/WorkflowDescriptorSpec.scala

```diff
@@ -76,13 +85,50 @@ class WorkflowDescriptorSpec extends FlatSpec with Matchers {
 
   }
 
+  def workflowFile(descriptor: WorkflowDescriptor, file: String): WdlFile = {
+    val path = descriptor.wfContext.root.toPath().resolve(file).toAbsolutePath
+    path.createIfNotExists() // TODO: Delete the files and directories!
```

Comment:
Yes. After the tests run, these specs do NOT delete the files. This is actually pretty common in a number of tests.